app/code/Magento/Sales/Test/Mftf/Test/StorefrontCustomerOrderDetailsPagerTest.xml

```diff
@@ -0,0 +1,286 @@
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct14">
+            <argument name="product" value="$$createProduct14$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct15">
+            <argument name="product" value="$$createProduct15$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct16">
+            <argument name="product" value="$$createProduct16$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct17">
+            <argument name="product" value="$$createProduct17$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct18">
+            <argument name="product" value="$$createProduct18$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct19">
+            <argument name="product" value="$$createProduct19$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddSimpleProductToShoppingCartActionGroup" stepKey="addProduct20">
+            <argument name="product" value="$$createProduct20$$"/>
+        </actionGroup>
+        <!-- 1 configurable -->
+        <actionGroup ref="StorefrontAddConfigurableProductToTheCartActionGroup" stepKey="addConfigurableToCart">
+            <argument name="urlKey" value="$createConfigProduct.custom_attributes[url_key]$"/>
+            <argument name="productAttribute" value="$$createConfigProductAttribute.default_value$$"/>
+            <argument name="productOption" value="$$getConfigAttributeSecondOption.label$$"/>
+            <argument name="qty" value="1"/>
+        </actionGroup>
+        <!-- 1 bundle -->
+        <actionGroup ref="StorefrontOpenProductPageActionGroup" stepKey="openBundleProduct">
+            <argument name="productUrl" value="$$createBundleProduct.custom_attributes[url_key]$$"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontAddBundleProductToTheCartActionGroup" stepKey="addBundleProductToCart">
+            <argument name="productName" value="$simpleProduct2.name$"/>
+            <argument name="quantity" value="1"/>
+        </actionGroup>
+        <!-- Place the order -->
+        <actionGroup ref="StorefrontOpenCheckoutPageActionGroup" stepKey="onCheckout"/>
+        <see userInput="22" selector="{{CheckoutOrderSummarySection.itemsQtyInCart}}" stepKey="see21Products"/>
+        <click selector="{{CheckoutShippingSection.firstShippingMethod}}" stepKey="selectFirstShippingMethod"/>
+        <actionGroup ref="StorefrontCheckoutClickNextOnShippingStepActionGroup" stepKey="clickNextButton"/>
+        <click selector="{{CheckoutPaymentSection.placeOrder}}" stepKey="placeOrder"/>
+        <waitForPageLoad stepKey="waitForSuccess"/>
```

Comment:
Please double-check if there is an action group for placing the order.

Reply:
Replaced with the existing one.